rosidl_typesupport_c/resource/srv__type_support.cpp.em

```diff
@@ -105,12 +118,92 @@ static const type_support_map_t _@(service.namespaced_type.name)_service_typesup
   &_@(service.namespaced_type.name)_service_typesupport_data.data[0],
 };
 
+
+@#TODO(ihasdapie): Import Postfixes
```

Comment:
I think this was done

Reply:
Done in https://github.com/ros2/rosidl_typesupport/pull/127/commits/f1bca7202581f53bfbacfb675b1c41d7ded85a47